relay-config/src/config.rs

```diff
@@ -193,8 +193,20 @@ trait ConfigObject: DeserializeOwned + Serialize {
             .with_context(|| ConfigError::file(ConfigErrorKind::CouldNotWriteFile, &path))?;
 
         match Self::format() {
-            ConfigFormat::Yaml => serde_yaml::to_writer(&mut f, self)
-                .with_context(|| ConfigError::file(ConfigErrorKind::CouldNotWriteFile, &path))?,
+            ConfigFormat::Yaml => {
+                f.write_all(b"# Please see the relevant documentation:\n")
```

Comment:
Could you pull these out into a single top-level constant (can be a `&str` if that's easier)?

Reply:
Example:

```rust
// put this somewhere top-level
static CONFIG_HEADER: &str = r###"# Please see the relevant documentation.
# Performance tuning: https://docs.sentry.io/product/relay/operating-guidelines/
# All config options: https://docs.sentry.io/product/relay/options/
"###;

f.write_all(CONFIG_HEADER.as_bytes())?
```

Reply:
Good suggestion, done in 348dece234459d5fbf914398677487aab0af91cb.

I spent too much time trying to do something like that, but I was using `const` and too many workarounds instead. So TIL the difference: https://github.com/rust-lang/rfcs/blob/master/text/0246-const-vs-static.md.

Reply:
It works the same with `const`, actually. See this playground:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=77507675c75105ce7f3166c7382a245b

Reply:
Hmmm, then I don't know what I was doing wrong 😕.